go/chat/attachment_httpsrv.go

```diff
@@ -287,14 +287,10 @@ func (r *AttachmentHTTPSrv) serveUnfurlVideoHostPage(ctx context.Context, w http
 					</script>
 				</head>
 				<body style="margin: 0px; background-color: rgba(0,0,0,0.05)">
-					<div style="display:flex; flex-direction: row; justify-content: flex-start">
-						<div style="flex: 1">
-						<video id="vid" onloadeddata="playVideo()" style="%s: 100%%; border-radius: 8px" src="%s" playsinline webkit-playsinline loop autoplay muted />
-						</div>
-					</div>
+					<video id="vid" onloadeddata="playVideo()" style="width: 100%%; height: 100%%; border-radius: 8px; object-fit:fill" src="%s" playsinline webkit-playsinline loop autoplay muted />
```

Comment:
this will be width/height 100% and will stretch to whatever the parent container is